issuetracker/src/main/java/org/openjdk/skara/issuetracker/Issue.java

```diff
@@ -104,6 +104,7 @@
 
     enum State {
         OPEN,
+        RESOLVED,
```

Comment:
Another option would be to simply map CLOSED to the Jira Resolved state as that's probably what we want to always use..

Reply:
Jira also supports renaming its states, so it's possible that we'll need to switch to setting State as a string to be interpreted as the particular Issue implementation sees fit.

Reply:
I think using `RESOLVED` makes sense for now :+1: